README.md

```diff
@@ -1,16 +1,334 @@
 # ActiveRecord Migrations
 
-1. How to create a new migration
-    a. when to use up and down
-    b. explain what change is
-2. how to create_table
-3. how to remove_table
-4. how to rename_table
-5. how to change_table
-6. how to remove_column
-7. how to add_column
-8. how to change_column
-9. how to add_foreign_key
-10. how to remove_foreign_key
-
-11. How to run a migration using Rake
+## Objectives
+
+1. Understand what migrations are and what they are used for.
+2. Learn how to create, modify and delete a table and columns in a table.
+3. Learn how to use Rake.
+
+### Creating Migrations
+
+Migrations are a convenient way for you to alter your database in a structured and organized manner. You could edit fragments of SQL by hand but you would then be responsible for telling other developers that they need to go and run them. You’d also have to keep track of which changes need to be run against the production machines next time you deploy.
+
+Migrations also allow you to describe these transformations using Ruby. The great thing about this is that it is database independent: you don’t need to worry about the precise syntax of CREATE TABLE any more than you worry about variations on SELECT * (you can drop down to raw SQL for database specific features). For example, you could use SQLite3 during development, but Postgres in production.
+
+Another way to think of migrations is like version control for your database. You might create a table, add some data to it, and then make some changes to it later on. By adding a new migration for each change you make to the database, you won't lose any data you don't want to, and you can easily revert changes.
+
+Executed migrations are tracked by ActiveRecord in your database, so they aren't used twice. Using the migrations system to apply the schema changes is easier than keeping track of the changes manually and executing them manually at the appropriate time.
+
```

Comment:
Add setup instructions here: "Fork and clone this repository, then run `bundle install`...